src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -318,7 +318,7 @@ void SolverInterfaceImpl:: initializeData ()
         suffix << _accessorName << ".init";
         exportMesh(suffix.str());
         if (context.triggerSolverPlot){
```

Comment:
We can also think about entirely removing this block:

```
if ( context.triggerSolverPlot ) {
...
}
```

**Don't forget to check whether `context.triggerSolverPlot` and corresponding API can be removed**